closed/OpenJ9.gmk

```diff
@@ -88,6 +88,8 @@ OPENJ9_NOTICE_FILE := openj9/longabout.html
 OPENJ9_NOTICE_FILE_RENAME := openj9-notices.html
 OPENJ9_REDIRECTOR := redirector/$(LIBRARY_PREFIX)jvm$(SHARED_LIBRARY_SUFFIX)
 
+OPENSSL_NOTICE_FILE := openssl-license
```

Comment:
openjdk and openj9 just have asingle file containing all the third party notices.  Calling the file openssl-license implies more files if more packages were to be added in future.  So why not have a single file (openj9-openjdk-notices.html?) and put the license and preamble in there, then if there are future packages the build won't need to be changed again?